src/views/List.jsx

```diff
@@ -1,6 +1,7 @@
 import { ListItem } from '../components';
-import { Navigate } from 'react-router-dom';
+import { Link, Navigate } from 'react-router-dom';
 import { useState, useEffect } from 'react';
+import { addItem } from '../api';
```

Comment:
Since we're not using it, I imagine we can clean up the component and remove this import